packages/qunit-dom/lib/assertions.ts

```diff
@@ -1,17 +1,17 @@
-import exists from './assertions/exists';
-import focused from './assertions/focused';
-import notFocused from './assertions/not-focused';
-import isChecked from './assertions/is-checked';
-import isNotChecked from './assertions/is-not-checked';
-import isRequired from './assertions/is-required';
-import isNotRequired from './assertions/is-not-required';
-import isValid from './assertions/is-valid';
-import isVisible from './assertions/is-visible';
-import isDisabled from './assertions/is-disabled';
-import matchesSelector from './assertions/matches-selector';
-import elementToString from './helpers/element-to-string';
-import collapseWhitespace from './helpers/collapse-whitespace';
-import { toArray } from './helpers/node-list';
+import exists from './assertions/exists.js';
```

Comment:
extensions are required by the "node16" moduleResolution option.
(some browser projects (and whole ecosystems) also use this)

Reply:
What about [moduleSuffixes](https://www.typescriptlang.org/tsconfig#moduleSuffixes) to make it automatic?